forum/views.py

```diff
@@ -211,3 +212,59 @@ def reply_thread(request, id):
 
         message.save()
         return redirect(message)
+
+def write_mail(request):
+    if request.method == 'GET':
+        message_id = request.GET.get('message')
+        message = get_object_or_404(Message, pk=message_id)
+
+        return render(request, "forum/write_mail.haml", {'errors' : [], "data": {
+            'title' : "",
+            'body' : "\n\n--------------------\n" + message.content
+        }})
+
+    if request.method == "POST":
+        errors = []
+        params = checkFields(request, errors)
+
+        if len(errors) == 0:
+            title = params["title"]
+            body = "De " + request.user.email + " :\n" + params["body"]
+            mail_from = request.user.email
+            mail_to = "jh.oscaredu@gmail.com" # A remplacer par le mail d'Oscar
```

Comment:
The real address is in settings.py in oscar/ folder. You can probably import that